lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -696,3 +696,41 @@ function dosomething_helpers_get_campaign_status($start_date, $end_date) {
     return 'closed';
   }
 }
+
+/**
+ * Gets the current run for a given campaign nid and user.
+ *
+ * @param int $nid
+ * @param object $user
+ *  Optional. Will default to global $user if not supplied
+ *
+ * @return node $run
```

Comment:
That's not the right way to declare return.
See example here https://github.com/drupal/drupal/blob/7.x/modules/node/node.module#L960-L961
and here https://www.drupal.org/node/1354#return.